bip-update-process.md

```diff
@@ -0,0 +1,646 @@
+toward improving the proposal based on community feedback, will be assigned a number by a BIP editor. The BIP editors
+should delay number assignment when they perceive a proposal being met with lack of interest: number
+assignment facilitates the distributed discussion of ideas, but before a proposal garners some interest in the bitcoin
+community, there is no need to refer to it by a number.
+
+Proposals are also not ready for number assignment if they duplicate efforts, disregard formatting rules, are too
+unfocused or too broad, fail to provide proper motivation, fail to address backwards compatibility, or fail to specify
+the feature clearly and completely. Reviewers and BIP editors should provide guidance on how the proposal may
+be improved to progress toward readiness. Pull requests that are proposing off-topic or unserious ideas or have
+stopped to make progress may be closed.
+
+When the proposal has been assigned a number and is complete, a BIP editor will merge it to the BIPs repository.
+After the BIP is merged to the repository, its main focus should no longer shift significantly, even while the author
+may continue to update the proposal as necessary in the git repository. Updates to merged documents by the author should also be
+submitted as pull requests.
+
+#### Proposed[^proposed]
+
+When the author is confident that their BIP represents a net improvement, is clear, comprehensive, has a working
+reference implementation (if applicable), and is ready for adoption by the bitcoin community, they may update the BIP’s
+status to Proposed to indicate that they recommend adoption or implementation of the BIP. Where applicable, the
+author must ensure that any proposed specification is solid, not unduly complicated, and definitive. Subsequently, the
+BIP’s content should only be adjusted in minor details, e.g. to improve language, clarify ambiguities, backfill
+omissions in the specification, add test vectors for edge cases, or address other issues discovered as the BIP is
+adopted.
+
+A Proposed BIP can only move to Active or Abandoned. Any necessary changes to the specification should be
+minimal and interfere as little as possible with ongoing adoption. If a Proposed BIP is found to need
+substantial functional changes, it may be preferable to move it to Abandoned[^new-BIP], and to start a new BIP with the changes
+instead. Otherwise, it could cause confusion as to what being compliant with the BIP means.
+
+#### Active
+
+A settled[^settled] BIP may be advanced to Active upon request by
+any community member with evidence that the idea described in the BIP is in active use. Such evidence includes
+for example: two or more projects having deployed support for the BIP in mainnet software releases, a soft fork
+proposal’s activation criteria having been met on the network, or rough consensus for the BIP having been demonstrated.
```

Comment:
```suggestion
A settled[^settled] BIP may be advanced to Active upon request by
any community member with evidence that the idea described in the BIP is in active use. Such evidence includes
for example: one or multiple projects having deployed support for the BIP in mainnet software releases, a soft fork
proposal’s activation criteria having been met on the network, or rough consensus for the BIP having been demonstrated.
```

I know this says "for example" but I'm sure there are cases where one project suffices (e.g., Bitcoin Core) and I want to save the future Editors from arguing about the number of implementations necessary to do the transition

Reply:
While I agree that in some cases one project might suffice, the example was deliberately suggesting that there should be at least two. Still thinking about it.

Reply:
Changed it to "at least one".